examples/src/pages/SimplePointcloud.tsx

```diff
@@ -182,7 +182,7 @@ export function SimplePointcloud() {
 
   return (
     <CanvasWrapper>
-      <Loader isLoading={isLoading} style={{ position: 'absolute' }}>
+      <Loader isLoading={loadingState.itemsLoaded !== loadingState.itemsRequested} style={{ position: 'absolute' }}>
```

Comment:
From my perspective, it's become harder to use, to be honest. 

Maybe we should have some simple loading state as we had which just indicates if something is loading at the moment? And in addition, have that one with progress? 

I just think that it's a very typical use-case to know is something loading or not and it should be solved as simple as possible. But drawing some things about progress is more advanced usage which shouldn't replace or make basic use-case more difficult. 

So I would suggest having another event for progress like `on('loadingProgressChanged', progressObject)` in addition to `loadingStateChanged` which returns boolean. Also, in that way we can keep API similar to the Cognite3DViewer which is not a bad thing.

Reply:
> counter goes from 88/89 to 1/1, or worse, 0/0

I think it's fine, to be honest. We don't provide entire progress functional here, right? Also, that information might be used in UI if you want to display it. Or if you don't you can implement that percentage stuff manually. Or we might extend loading state object to have that. E.g. every event will have:

```ts
export type LoadingState = {
  itemsLoaded: number;
  itemsRequested: number;
  progress: number; // 0 - 100 based on `itemsLoaded / itemsRequested`. It's a dummy but a convenient thing. 
};
```

Reply:
@larsmoa @j-bjorne gently nudging you to agree or disagree with that suggestion.

Reply:
hehe, just reloaded the page and saw your comments. I think we went for
```
export interface LoadingState {
  isLoading: boolean;
  itemsLoaded: number;
  itemsRequested: number;
}
```

Reply:
That way you can use only the isLoading if you want to, and then if you want the details you use itemsLoaded and itemsRequested